exporter/exporterhelper/common.go

```diff
@@ -154,6 +156,14 @@ func WithQueue(queueSettings QueueSettings) ExporterOption {
 	}
 }
 
+// WithResourceToLabelConversion overrides the default ResourceToLabelSettings for an exporter.
+// The default ResourceToLabelSettings is to disable resource attributes to labels conversion.
+func WithResourceToLabelConversion(resourceToLabelSettings ResourceToLabelSettings) ExporterOption {
```

Comment:
I don't see where you plugin the consumer, do I miss anything?

Reply:
Hi @bogdandrutu thanks for your quick feedback. I found your 2nd suggestion pretty simple and easily plug-able (use the helper function directly instead of writing a consumer). In my last commit, I removed the consumer and called the `convertResourceToLabels()` directly from the existing metricshelper's `ConsumeMetrics()`. 

Expecting your second review. If it seems good, I will send another PR with README update and other minor suggestion.

Test:
I tested manually with `logging` exporter. Everything is working fine. Customer can configure it from ot-config file.